electrical/readme.MD

```diff
@@ -0,0 +1,261 @@
+
+**Table 5. J8 Pins and Wire Colors**
+
+Control Board PCB J8 pin | TTL Serial cable wire color
+------------------------ | ---------------------------
+GND | Black
+N/A | Not Connected
+5V | Red
+TxD | Green
+RxD | White
+N/A | Not Connected
+
+![USB-serial](../images/electronics/usb_ttl_serial.png)
+
+  **Figure 5. USB / TTL Serial Cable Connection**
+  
+# 4. Connecting Motors and Encoders
+Next, we will connect the motors and encoders to the RoboClaw motor controllers. For each of the 10 motors you will need to connect motor power as well as the encoders. This means that there will be close to 60 wires traveling between the motor controllers and the motors/encoders, so it is very important to try to keep your wires organized and bundled together well. Below is some information about strain relief and using wire braid in your electronics systems. See these links and Figures 6 and 7 for examples.
+
+* [Techopedia Strain Relief](https://www.techopedia.com/definition/2301/strain-relief) 
+* [YouTube Wire Braid](https://www.youtube.com/watch?v=FeCs98TSsYQ)
+
+![](Pictures/Routing/route1.PNG)![](../images/electronics/route2.png)![](Pictures/Routing/route3.PNG)
+
+**Figure 6. Motor Wire Routing**
+    
+An example of how we routed our wiring is shown by Figure 6. It is important to give strain relief in the cable as you are routing it through, as well as to give extra wiring for the locations that can move. In addition, Figure 7 shows how we routed the wires through the rocker-bogie, and then from the rocker-bogie into the main body. Try to pick locations that reduce strain on the wires as the robot legs move when you are routing wires into the body.
+
+![](../images/electronics/route5.png)
+
+**Figure 7. Serial signal routing**
+
+**Table 6. Necessary Parts**
+
+Item | Ref | Qty | Image
+---- | --- | --- | ----- 
+20AWG Wire (Black) | X2 | N/A | <p align="center"> <img src="../images/components/wiring/X2.png" width="25%"> </p>
+20AWG Wire (Red) | X1 | N/A | <p align="center"> <img src="../images/components/wiring/X1.png" width="25%"> </p> 
+30AWG Wire (White) | X3 | N/A | <p align="center"> <img src="../images/components/wiring/X3.png" width="25%"> </p> 
+30AWG Wire (Blue) | X4 | N/A | <p align="center"> <img src="../images/components/wiring/X4.png" width="25%"> </p> 
+30AWG Wire (Yellow) | X5 | N/A | <p align="center"> <img src="../images/components/wiring/X5.png" width="25%"> </p> 
+30AWG Wire (Green) | X6 | N/A | <p align="center"> <img src="../images/components/wiring/X6.png" width="25%"> </p> 
+3 Pin Micro Connector | E40 | 4 | <p align="center"> <img src="../images/components/electronics/E40.png" width="25%"> </p>
+Wire Braid | E42 | N/A | <p align="center"> <img src="../images/components/electronics/E42.png" width="25%"> </p>
+Heat Shrink Tubing | E44 | 1 | <p align="center"> <img src="../images/components/electronics/E44.png" width="25%"> </p>
```

Comment:
Make this table 2 columns, refer to https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md

Reply:
done